go/cmd/vttablet/status.go

```diff
@@ -85,82 +77,18 @@ var (
       <a href="/healthz">Health Check</a></br>
       <a href="/debug/health">Query Service Health Check</a></br>
       <a href="/streamqueryz">Current Stream Queries</a></br>
+      <a href="/debug/status_details">JSON Status Details</a></br>
     </td>
   </tr>
 </table>
-`
-
-	// healthTemplate is just about the tablet health
-	healthTemplate = `
-<div style="font-size: x-large">Current status: <span style="padding-left: 0.5em; padding-right: 0.5em; padding-bottom: 0.5ex; padding-top: 0.5ex;" class="{{.CurrentClass}}">{{.CurrentHTML}}</span></div>
-<p>Polling health information from {{github_com_vitessio_vitess_health_html_name}}. ({{.Config}})</p>
-<h2>Health History</h2>
-<table>
-  <tr>
-    <th class="time">Time</th>
-    <th>Healthcheck Result</th>
-  </tr>
-  {{range .Records}}
-  <tr class="{{.Class}}">
-    <td class="time">{{.Time.Format "Jan 2, 2006 at 15:04:05 (MST)"}}</td>
-    <td>{{.HTML}}</td>
-  </tr>
-  {{end}}
-</table>
-<dl style="font-size: small;">
-  <dt><span class="healthy">healthy</span></dt>
-  <dd>serving traffic.</dd>
-
-  <dt><span class="unhappy">unhappy</span></dt>
-  <dd>will serve traffic only if there are no fully healthy tablets.</dd>
-
-  <dt><span class="unhealthy">unhealthy</span></dt>
-  <dd>will not serve traffic.</dd>
-</dl>
 `
 )
 
-type healthStatus struct {
-	Records []interface{}
-	Config  template.HTML
-	current *tabletmanager.HealthRecord
-}
-
-func (hs *healthStatus) CurrentClass() string {
-	if hs.current != nil {
-		return hs.current.Class()
-	}
-	return "unknown"
-}
-
-func (hs *healthStatus) CurrentHTML() template.HTML {
-	if hs.current != nil {
-		return hs.current.HTML()
-	}
-	return template.HTML("unknown")
-}
-
-func healthHTMLName() template.HTML {
-	return health.DefaultAggregator.HTMLName()
-}
-
 func addStatusParts(qsc tabletserver.Controller) {
 	servenv.AddStatusPart("Tablet", tabletTemplate, func() interface{} {
 		return map[string]interface{}{
-			"Tablet":               topo.NewTabletInfo(tm.Tablet(), nil),
-			"BlacklistedTables":    tm.BlacklistedTables(),
-			"DisallowQueryService": tm.DisallowQueryService(),
-		}
-	})
-	servenv.AddStatusFuncs(template.FuncMap{
-		"github_com_vitessio_vitess_health_html_name": healthHTMLName,
-	})
-	servenv.AddStatusPart("Health", healthTemplate, func() interface{} {
```

Comment:
What is the reason for deleting this?

Reply:
This is taken over by tabletserver's status func, which also publishes additional details on why a tablet is not serving.